app/views/validators/_table.html.erb

```diff
@@ -166,6 +167,16 @@
               </div>
             </div>
             <div>
+              <% if validator.score.software_client == "firedancer" %>
+                <div class="mt-2 mt-lg-0 d-none d-lg-inline-block">
+                  <%= link_to validators_path(network: params[:network], q: validator.score.software_version) do %>
+                    <%= image_tag("firedancer.svg", class: 'img-xs me-1', title: "Show Firedancer validators", alt: "F") %>
+                  <% end %>
+                </div>
+              <% elsif validator.score.software_client == "agave" %>
+                <!-- TODO puts agave logo here -->
```

Comment:
perhaps it should just be a default solana logo ?

Reply:
Brian contacted Anza and they will send us the logo